reference/constraints/Email.rst

```diff
@@ -133,11 +130,6 @@ The default value used by this option is set in the
 :ref:`framework.validation.email_validation_mode <reference-validation-email_validation_mode>`
 configuration option.
 
-.. deprecated:: 6.2
-
-    The ``loose`` value is deprecated since Symfony 6.2. Starting from
-    Symfony 7.0, the default value of this option will be ``html5``.
-
```

Comment:
This needs to be reverted, it will be removed in the 7.0 branch

Reply:
You right, it was one of my first contribution and I hadn't paid attention about that.

Reply:
No worries, congrats for this first one! :slightly_smiling_face: